packages/ui/src/constants/contracts.ts

```diff
@@ -0,0 +1,81 @@
+import { Address } from "viem";
+import { NetworkId } from "./networks";
+
+export const paymentReceiverContractsByNetworkId: Record<NetworkId, Address> = {
+  [NetworkId.TAURUS]: "0x0000000000000000000000000000000000000000",
```

Comment:
Do you mean Chronos here? (And in the next file.)

Reply:
Marked for removal, in following PR